docs/test-set.md

```diff
@@ -0,0 +1,86 @@
+@module {function} can-define-validate-validatejs/DefineMap.prototype/testSet testSet
+@parent can-define-validate-validatejs/DefineMap.prototype
+
+Tests value changes against constraints. The map is serialized then updated based
```

Comment:
There's too much here.  Move the technical detail about how it works to within the signatures.